### PR TITLE
[Feedback] Add preload spinner

### PR DESCRIPTION
**Pull request by nerdlet**
##### What does this PR do?
=> Add preloader before react DOM is rendered on the site
##### Screenshots?
Current load: 
currentPreload

Fix: 
preload

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -10,21 +10,30 @@ module.exports = {
...
-        display: `standalone`,
-      },
+        display: `standalone`
+      }
     },
+    {
+      resolve: `gatsby-plugin-nprogress`,
+      options: {
+        // Setting a color is optional.
+        color: `tomato`,
```
Comment (kilemensi) [inline gatsby-config.js]: `tomato`? Is this the name of SeaSensors main/primary color? Otherwise switch to that.

File: src/html.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {headComponents}
+      </head>
+      <body {...bodyAttributes}>
+        {preBodyComponents}
+        <noscript key="noscript" id="gatsby-noscript">
+          This app works best with JavaScript enabled.
+        </noscript>
+        <div className="preloader"></div>
```
Comment (kilemensi) [inline src/html.js:28]: I'm assuming we're adding this file _just_ to insert this `div` here. Can't we add the div in the `Page` component instead: https://github.com/CodeForAfrica/sensors.AFRICA-SeaSensors/blob/e414a8b662e94765fa10391dd6ccd88676eecc2c/src/components/core/Page.js#L10

All pages already use this component so if it works here, it should be available in all pages.

**Review by kilemensi — CHANGES_REQUESTED** [review]
Better that the arrow @nerdlet :+1: but we still need to figure out why the pages are so *heavy* (not this sprint since we should be focusing on HURUmap UI).

Restore the `yarn.lock` file you deleted. It's a very important file for reproducible build/deployments.

File: src/html.js (hunk 2 above)
Comment (nerdlet) [reply src/html.js:28]: @kilemensi Maybe a small update on this...putting the` preload ` page.js loads it at the same time with everything else on the page... And we need to catch the `::before ` the page loads for us to load the preloader spinner. Not sure if you have any suggestions on the way forward?

**Comment by kilemensi** [comment]
Good catch @nerdlet. Lets just add the HTML file for now then.

**Comment by nerdlet** [comment]
@kilemensi Done.

**Pull request merged**